debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -508,8 +508,13 @@ protected Object convertPoint(Column column, Field fieldDefn, Object data){
         if (data == null) {
             data = fieldDefn.schema().defaultValue();
         }
-
+        
```

Comment:
Could you configure your IDE so it removes any trailing whitespace? E.g. there's a "Save action" for that in Eclipse, I'd expect similar options to exist in other IDEs. Avoiding this kind of noise makes it easier for all of us to work with the code, review changes etc. Thanks!

No need to change this very one, though. I'll take care of it prior to merging.